pythainlp/util/normalize.py

```diff
@@ -299,4 +299,4 @@ def maiyamok(sent: Union[str, List[str]]) -> List[str]:
             i += 1
         _list_word.append(text)
         i += 1
-    return _list_word
+    return _list_word
```

Comment:
Please add newline to the end of file.

Refer to this POSIX standard
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

See some examples for illustration at https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Oh, my mistake.
Fixed!